packages/mdc-drawer/README.md

```diff
@@ -283,15 +359,29 @@ Method Signature | Description
 `focusActiveNavigationItem() => void` | Focuses the active / selected navigation item.
 `notifyClosed() => void` | Emits the `MDCDrawer:close` event.
 `notifyOpen() => void` | Emits the `MDCDrawer:open` event.
+`trapFocusOnSurface() => void` | Traps focus on surface and focuses the active navigation element.
+`untrapFocusOnSurface() => void` | Removes trap focus from surface and resumes focus to nav button.
 
-### Foundations: `MDCDismissibleDrawerFoundation`
+### Foundations
+
+#### `MDCDismissibleDrawerFoundation`
 
 Method Signature | Description
 --- | ---
 `open() => void` | Opens the drawer from the closed state.
 `close() => void` | Closes the drawer from the open state.
+`opened() => void` | Abstract method which gets called when drawer finished opening.
```

Comment:
I'd use the word "Extensible" for these rather than "Abstract".

Shouldn't these _not_ technically be `@abstract` on dismissible drawer? They're just no-ops for it. Presence of abstract methods normally suggests the class can't be instantiated, doesn't it?